extension/scripts/Selector/SelectorTable.js

```diff
@@ -22,7 +22,7 @@ var SelectorTable = {
         var columns = {};
         var headerRowSelector = this.getTableHeaderRowSelector();
         var $headerRow = $table.find(headerRowSelector);
-        var isVerticalRow = false;
+        var isVerticalRow = this.verticalTable;
 
         if ($headerRow.length > 0) {
```

Comment:
So many nested if expressions is hard to understand. Whe should discuss how to refactor it.